shardingsphere-features/shardingsphere-shadow/shardingsphere-shadow-core/src/main/java/org/apache/shardingsphere/shadow/algorithm/shadow/hint/SimpleHintShadowAlgorithm.java

```diff
@@ -53,8 +65,17 @@ public boolean isShadow(final Collection<String> shadowTableNames, final Precise
         if (ShadowOperationType.HINT_MATCH != noteShadowValue.getShadowOperationType() && !shadowTableNames.contains(noteShadowValue.getLogicTableName())) {
             return false;
         }
-        return ShadowHintExtractor.extractSimpleHint(noteShadowValue.getValue())
-                .filter(optional -> props.entrySet().stream().allMatch(entry -> Objects.equals(entry.getValue(), optional.get(String.valueOf(entry.getKey()))))).isPresent();
+        return ShadowHintExtractor.extractSimpleHint(noteShadowValue.getValue()).filter(this::containsHint).isPresent();
+    }
+    
+    private boolean containsHint(final Map<String, String> preciseHint) {
+        for (Map.Entry<String, String> entry : simpleHint.entrySet()) {
```

Comment:
Please do not use `Map.Entry`, just put Map.Entry into import list

Reply:
I will fix it.